src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -90,11 +90,13 @@ class ChaptersButton extends TextTrackButton {
     let menu = this.menu;
     if (menu === undefined) {
       menu = new Menu(this.player_);
-      menu.contentEl().appendChild(Dom.createEl('li', {
+      let title = Dom.createEl('li', {
```

Comment:
1/. Since this is a new Menu, what's the situation where it can have children already?

2/. meun-button.js already has this mechanism to add a title (using options.title) which doesn't seem to be used elsewhere - why isn't that mecanism used here?